packages/eslint-config/src/flat/configs/vue.ts

```diff
@@ -7,6 +7,47 @@ import type { FlatConfigItem } from 'eslint-flat-config-utils'
 import type { NuxtESLintConfigOptions } from '../types'
 import { removeUndefined, resolveOptions } from '../utils'
 
+const INLINE_ELEMENTS = [
```

Comment:
Let's have a comment link to the source so we remember to keep in sync, or import from .json later